### DISCUSSION TITLE
【寻物启示】震惊!!!!!!这种书籍竟被那位封禁

### DISCUSSION THREAD

**Discussion by tingsha1935** [share121/inter-knot#2267] [其他]
![1788b37ac79d0f61d07d15d025cc211](https://github.com/user-attachments/assets/0c966195-f2de-4d6a-baa6-669318206c35)
我好不容易寻得一本治愈的书籍，主人公十分善良，世人皆称为“大爱仙尊”，如此制作精良文笔上乘的小说竟被【北京作家协会副主席】9转亡妻蛊为本命蛊的糖加3勺使用10转净网蛊封杀，实属不该
麻烦帮我寻得【大爱仙尊-残卷】
奖励：
练习时长达2.5年的鸡*1
丁尼2.5万
菲林25

**Comment by God-Arch** [comment]
不是停更了吗？我这有无删减的

**Comment by DYSQ666** [comment]
独帜入渊深未知，
[身似浮萍命难持,
千州皆朝归海处，
一苇青拨戏浪巅
惊鸿四散鱼逃尽，
唯有残帆傲此间，
待到天开云雾散，
负手直望笑苍天！
我清楚的知道，人与人的路都是不可复制的，我走在我自己的人生路上，哪怕路途的风雨太大，大到我步履维艰，哪怕荆棘丛生，刺得我伤痕遍布，我也仍旧痴痴笑笑，我体会此中滋味，呵呵呵，我相信独有的小船终有一天会看到两岸群山青翠虫鸟同鸣……

**Comment by huogou114514** [comment]
"方源，乖乖地交出春秋蝉，我给你个痛快！"
"方老魔，你不要妄图反抗了，今日我们正道各大派联合起来，就是要踏破你的魔窟。这里早已经布下天罗地网，这次你必定身首异处！"
"方源你个该死的魔头，你为了练成春秋蝉，杀了千万人的性命。你已经犯下了滔天的罪孽，罪无可恕，罄竹难书！"
"魔头，三百年前你侮辱了我，夺走了我的清白之身，杀光我全家，诛了我的九族。从那刻起，我恨不得吃你肉，喝你的血！今天，我要让你生不如死！!"
方源一身残破的碧绿大袍，披头散发，浑身浴血，环顾四周。
山风吹得血袍飘荡，如战旗般嚯嚯作响。
鲜红的血液，从身上数百道伤口向外涌着。只是站着一会儿，方源脚下已经积了一大滩的血水。
群敌环伺，早已经没有了生
路。
大局已定，今日必死无疑。
方源对局势洞若观火，不过即便死亡将临，他仍旧是面不改色，神情平淡。
他目光幽幽，如古井深潭一般，一如既往的深不见底。
围攻他的正道群雄，不是堂堂一派之长者尊贵，就是名动四方之少年英豪。此时牢牢包围着方源有的在咆哮，有的在冷笑，有的双眼眯起闪着警惕的光，有的捂着伤口恐惧地望着。
他们没有动手，都忌惮着方源的临死反扑。
就这样紧张地对峙了三个时辰，夕阳西下，落日的余晖将山边的晚霞点燃，一时间绚烂如火。
一直静如雕塑的方源，慢慢转身。
群雄顿时一阵骚动，齐齐后退一大步。
此刻，方源脚下的灰白山石，早已经被鲜血染成了暗红。因为失血过多而显得苍白的脸，被晚霞映照着，忽然增添了一份嫣然之光彩。
看着这青山落日．方源轻声一笑："青山落日，秋月春风。当真是朝如青丝暮成雪，是非成败转头空。"
说这话的时候，眼前忽的就浮现出前世地球上的种种。
他本是地球上的华夏学子，机缘巧合穿越到这方世界。辗转颠簸三百年，纵横世间两百余年，五百多年光阴悠悠，却是晃眼即逝。
深埋在心底的许多记忆，在此刻鲜活起来，栩栩如生地在眼前回现着。
"终究是失败了呀。"方源心中叹着，有些感慨，却并不后悔。
这种结果，他也早有预见。当初选择时，就有了心理准备。
所谓魔道，就是不修善果，杀人放火。天地不容，举世皆敌，还要纵情纵横。
"若是刚炼成的春秋蝉有效，来生还是要做邪魔！"这般想着，方源情不自禁放声大笑。
"老魔，你笑什么？"
"大家小心，魔头死到临头要
反扑了！"
"快快交出春秋蝉！!"
群雄逼迫而来，恰在这时，轰的一声，方源悍然自爆。
春雨绵绵，悄无声息地滋润着
青茅山。
夜已经深了，丝丝凉风吹拂着
细雨。
青茅山却不黑暗，从山腰至山脚，闪着许多莹莹的微光，好像是披着一条灿烂的光带。
这些光来源于一座座高脚吊楼，虽称不上万家灯火，却也有数千的规模。
正是坐落在青茅山的古月山寨，给广袤幽静的山峦增添了一份浓郁的人烟气息。
我只有这些残卷了，剩下的道友另寻他法吧

**Reply by DreamEndn** [reply]
这位仙友，我已经炼成，十转！无删减大爱仙尊蛊了！

**Comment by DreamEndn** [comment]
春秋蝉：很简单，我成尊不就是了
方源：？
乐土：？
陆畏因：？？
巨阳、星宿：？？？？？
红莲：？？？？？？？？？
只听它口中忽吟：
了了了知了了知，知了知知了知知。
知了知知知知了，了了知知了知知。
知知了知知了了，知知知了了知了。
知知了了知知了，了了了知知了知！

我用蓝奏浏览器分享了[大爱仙尊无删减], 下载链接:https://wwp.lanzoup.com/iL0DZ27swc8f  提取码 : daxz

**Comment by LuAn-afl** [comment]
太好了是大爱仙尊大人我们死定了<img src="https://bbs-static.miyoushe.com/static/2023/11/07/722fc5f2e631e5c02184afdc15c2209f_7114079448845986183.png" alt="藿藿 抓狂" width="75" >
